### PR TITLE
Parvathy | Update docker image tags to latest

### PR DESCRIPTION
**Pull request by parvathy00**
jira card -> https://bahmni.atlassian.net/browse/BAH-3068

In this PR, the IMAGE_TAGS in .env file in bahmni-lite is updated to latest.

### CODE REVIEW

**Review by N0-man — CHANGES_REQUESTED** [review]
.env is the contract that represent the version of individual apps we are releasing with docker bahmni-lite. Increment with specific version only if the respective app is being upgraded and release candidate for next increment. You can use latest for your local development .env file for testing...

**Comment by mohan-13** [comment]
> .env is the contract that represent the version of individual apps we are releasing with docker bahmni-lite. Increment with specific version only if the respective app is being upgraded and release candidate for next increment. You can use latest for your local development .env file for testing...

We are making the tags to latest in master branch of bahmni-docker to make it default for development setup or for people who want to try to the latest build of Bahmni Lite images. It becomes hard to update individual image tags to latest as different images get updated at different point in time.

**Comment by N0-man** [comment]
you could create and commit `.env.dev` file instead that points to latest for development and quick bootstrap? `.env` cloud still represent the snapshot of individual application versions we are releasing with a release of bahmni-lite (remember that we wouldnt bump the version of all the apps with bahmni-lite release). We need to then update the docs and perhaps `runBahmni.sh` to leverage the right environment file - with default being `.env`

```
#!default .env as value if not passed via cli
file=$1   

docker compose --env-file $file up
```

**Comment by gsluthra** [comment]
Will be good to have `.env-production` file or something similar, for people who want to run Bahmni with specific LiteV1 released version. This way our existing automation and build processes won't need modifications, and most people currently will want `.env` (with latest). We can document in readme, that if u want liteV1 or run docker in production, please rename the `.env-production` to `.env`, or run with `--env-file` flag.

Alternatively, we can also have `docker-compose-production.yml` file. 

See: https://stackoverflow.com/questions/47039538/docker-compose-for-production-and-development

**Comment by mohan-13** [comment]
Creating a new .env file would increase the maintenance overhead. For every new variable addition/modification it needs to be done in both the files . Now that we have two flavours of Bahmni eventually it would become  same change across 4 files for any common variable. 
Also one more thing to note is that when we spin up an environment by specifying --env-file option then every-time the option needs to be passed on with subsequent docker compose up command. Or else docker compose will automatically recreate containers with the config in default file (ie. .env). This would not be a major problem in dev environments, so we can have a env.dev which will point to latest tags. For production/litev1 we still use .env

**Comment by N0-man** [comment]
@mohan-13 I believe that this requirement of having everything `latest` is coming more as a convenience for the active development team AND NOT really from the implementers or prospects who wants to quickly try it (bootstrap)?? Because  for the later (implementers or prospects) it would not be appropriate for them to run bahmni-lite with `latest` since `latest = wip / in-complete / non-released / not-fully-tested instance`

**Comment by gsluthra** [comment]
Let's please make a decision so that people are UNBLOCKED and environments are up-to-date. Here is my decision:

1. Create an `.env.dev` for both `bahmni-lite` and `bahmni-standard`, which should point to `latest` for all relevant bahmni images.
2. Leave `.env` as-is, to continue to point to LiteV1 stable images. This will ensure that implementors and evaluators don't need to perform any additional steps to bring up a stable version of Bahmni. No documentation updates needed for them.
3. Modify our environment startup scripts to either replace `.env` with `.env.dev` or use the `--env-file` option.
4. Update the README.md of this repo to indicate that if you want the latest of all images, which could be a bit unstable, then use the `.env.dev`
5. For the time being, whenever there is a PR which changes .env, we will need to ensure all 4 files have the same relevant change. If not, we will know about it. We can write a simple build-time script that will check the number of KEYS are exactly the same in both `.env` and `.env.dev` for lite or standard, to ensure someone doesn't add a special KEY in just one of the two files. This can be a later task, when we think this issue is an ACTUAL issue to solve.

cc: @parvathy00 @mohan-13

File: .github/workflows/bahmni_lite_docker_env.yaml [hunk 1]
```diff
@@ -42,9 +42,9 @@ jobs:
       - name: Create/Update Containers
         run: |
           if [ "$UPDATE_BAHMNI_OPTION" == "update_without_removing_volumes" ]; then
-            command="docker compose up -d"
+            command="docker compose --env-file .env.dev up -d"
           elif [ "$UPDATE_BAHMNI_OPTION" == "update_and_remove_volumes" ]; then
-            command="docker compose down -v && docker compose up -d"
+            command="docker compose down -v && docker compose --env-file .env.dev up -d"
```
Comment (mohan-13) [inline .github/workflows/bahmni_lite_docker_env.yaml:47]: --env-file option should be added for down command as well

Comment (mohan-13) [reply .github/workflows/bahmni_lite_docker_env.yaml:47]: Also add --env-file option at line 63 as well for the docker compose pull command

Comment (parvathy00) [reply .github/workflows/bahmni_lite_docker_env.yaml:47]: sure Mohan, will add it

**Review by mohan-13 — APPROVED** [review]

**Pull request merged**